week-1-practise/level-1/9-function-output-2.js

```diff
@@ -1,14 +1,22 @@
 function s(w1, w2) {
-    return w1.concat(w2);
+  return w1.concat(w2);
 }
 
-var result = s('code', 'your');
+var result = s("code", "your");
 console.log(result);
 
 // Answer these questions:
 // - What does this program do? Run it and see, or Google some of the methods used ot understand them.
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
-// - What does the function return?
-// - How can this function be improved?
\ No newline at end of file
+// it join two strings togather
+// - How many parameters does the function take?It took two
+// - What is the function name? s
+// - Where is the function called? with what parameters? In line no.5 with two parameters 'code' and 'your'
+// - What does the function return? codeyour
+// - How can this function be improved? we could change the name of the function and paramerers so it shows
+// it will be easy to understand the function
+/*function string(str1, str2) {
```

Comment:
Very good answer! The name is certainly the biggest problem with this function at the moment and your reason you gave is exactly right.

In coding, the word 'string' has a special meaning of text data, so maybe `string` is still not the best name it could have.

Reply:
yes you are right may be using  
connect 
as a name  will  be better